Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'plek', '~> 1.10'
 gem 'airbrake', '~> 4.2.1'
 gem 'govuk_admin_template', '~> 3.0.0'
 gem 'gds-sso', '~> 11.0.0'
+gem 'gds-api-adapters'
```

Comment:
Whenever we add a dependency we should provide an explicit version: https://github.com/alphagov/styleguides/blob/master/ruby.md#gemfile